packages/vetrina/index.js

```diff
@@ -1,4 +1,4 @@
-require("./build/Vetrina.css");
```

Comment:
Delete rather than commenting out?

Reply:
I actually think it's supposed to be there, I commented it out when I thought that we could altogether delete Vetrina less file, but then as it is needed for dark mode, I guess this should not be commented out or deleted